pkg/cli/admin/catalog/mirror.go

```diff
@@ -314,6 +317,10 @@ func (o *MirrorCatalogOptions) Complete(cmd *cobra.Command, args []string) error
 		return "", fmt.Errorf("no database file found in %s", e.Mappings[0].To)
 	}
 	o.DatabaseExtractor = extractor
+
+	if o.IcspScope == "" {
+		o.IcspScope = "repository"
```

Comment:
set the default when creating options https://github.com/openshift/oc/pull/485/files#diff-7db656e03ae09bb1778bb0a8cde7fd18R96

Reply:
Done.